game/domain/body.lua

```diff
@@ -440,4 +443,14 @@ function Body:getKiller()
   return self.killer
 end
 
+--Dialogue methods
+
+function Body:getDialogue()
+  return self.dialogue
+end
+
+function Body:setDialogue(dialogue)
+  self.dialogue = dialogue
+end
+
```

Comment:
While I understand why it would be in the Body class, I'm thinking that this class is becoming bloated because of it.

See also: actor class.

See also: https://refactoring.guru/smells/large-class

Reply:
I agree. But this PR is not meant to refactor Body, I'll leave it to another PR.